includes/widgets/common.php

```diff
@@ -153,55 +153,50 @@ protected function register_controls() {
 		);
 
 		$this->add_responsive_control(
-			'_z_index',
+			'_element_custom_width',
 			[
-				'label' => esc_html__( 'Z-Index', 'elementor' ),
-				'type' => Controls_Manager::NUMBER,
```

Comment:
All of the mess here is because of moving the controls around in order to re-organize them.